webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3107,6 +3107,22 @@ var userOverride = {
                             'aria-label',
                             preLeadCollection.field
                         );
+                        kmChatInput.setAttribute(
+                            'pattern',
+                            "^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,7}$"
+                        );
+                        kmChatInput.setAttribute(
+                            'title',
+                            ''
+                        );
+                        kmChatInput.setAttribute(
+                            'oninvalid',
+                            "setCustomValidity('Please enter a valid email address')"
```

Comment:
use MCK_LABELS
https://docs.kommunicate.io/docs/web-localization

Reply:
Does this doc also needs to be updated?
have updated in the code.